2022-01-09 18:51:17,255 INFO    MainThread:20903 [internal.py:wandb_internal():90] W&B internal server running at pid: 20903, started at: 2022-01-09 18:51:17.255303
2022-01-09 18:51:17,258 INFO    WriterThread:20903 [datastore.py:open_for_write():77] open: /content/tbcnn/wandb/offline-run-20220109_185116-1kd6mc18/run-1kd6mc18.wandb
2022-01-09 18:51:17,259 DEBUG   HandlerThread:20903 [handler.py:handle_request():130] handle_request: run_start
2022-01-09 18:51:19,993 DEBUG   HandlerThread:20903 [meta.py:__init__():40] meta init
2022-01-09 18:51:19,994 DEBUG   HandlerThread:20903 [meta.py:__init__():54] meta init done
2022-01-09 18:51:19,994 DEBUG   HandlerThread:20903 [meta.py:probe():214] probe
2022-01-09 18:51:19,997 DEBUG   HandlerThread:20903 [meta.py:_setup_git():204] setup git
2022-01-09 18:51:20,023 DEBUG   HandlerThread:20903 [meta.py:_setup_git():211] setup git done
2022-01-09 18:51:20,023 DEBUG   HandlerThread:20903 [meta.py:_save_pip():58] save pip
2022-01-09 18:51:20,024 DEBUG   HandlerThread:20903 [meta.py:_save_pip():72] save pip done
2022-01-09 18:51:20,024 DEBUG   HandlerThread:20903 [meta.py:probe():252] probe done
2022-01-09 18:52:38,379 WARNING MainThread:20903 [internal.py:is_dead():367] Internal process exiting, parent pid 20893 disappeared
2022-01-09 18:52:38,380 ERROR   MainThread:20903 [internal.py:wandb_internal():142] Internal process shutdown.
2022-01-09 18:52:39,173 INFO    WriterThread:20903 [datastore.py:close():281] close: /content/tbcnn/wandb/offline-run-20220109_185116-1kd6mc18/run-1kd6mc18.wandb
2022-01-09 18:52:39,190 INFO    HandlerThread:20903 [handler.py:finish():731] shutting down handler
2022-01-09 18:52:39,312 INFO    SenderThread:20903 [sender.py:finish():1070] shutting down sender
2022-01-09 18:52:39,325 INFO    MainThread:20903 [internal.py:handle_exit():77] Internal process exited
